sdk/core/core-http/lib/fetchHttpClient.ts

```diff
@@ -179,12 +174,7 @@ export abstract class FetchHttpClient implements HttpClient {
           httpRequest
         );
       } else if (fetchError.type === "aborted") {
-        throw new RestError(
-          "The request was aborted",
-          RestError.REQUEST_ABORTED_ERROR,
```

Comment:
This static readonly property `REQUEST_ABORTED_ERROR` was probably used only for this purpose. Now that we intend to use the `AbortError` in its place, we should remove this static property too.

Reply:
Done.